src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    RED,
+    BLUE,
+    BLACK,
+    WHITE,
+}
```

Comment:
Enum values should be named using uppercase letters. In Java, it is a common practice to use all uppercase letters for enum constants, which you have followed correctly here. However, the enum itself should be named in singular form, `Color` rather than `Colors`, as each constant represents a single color.